### DISCUSSION TITLE
After changing MySQL to MariaDB, updating Apache and PHP - laragon slow. How to improve?

### DISCUSSION THREAD

**Discussion by AWqxKAWERbXo** [leokhoa/laragon#872] [Q&A] [answered]
What I liked most about laragon was/is the performance out of the box. But migrating to MariaDB and updating Apache and PHP (<https://github.com/leokhoa/laragon/wiki> made Laragon perform as slow as XAMPP.

Laragon is advertising that useful defaults are configured for everything, so I guess I missed something.

I already had a look into https://github.com/leokhoa/laragon/discussions/848 but that's a diffrent problem (it's not drupal). Also, it used to be fast before changing Apache, MySQL->MariaDB.

Can someone provide me additional information or "the missing piece" what's needed to make it fast again?

**Comment by ppfoong** [comment]
Check your my.ini config

**Reply by AWqxKAWERbXo** [reply]
Checking for...?

**Reply by ppfoong** [reply, marked as answer]
Add these lines into my.ini. It is for computer with 8GB RAM. If you have more RAM, you can increase the values accordingly.

`
[mysqld]
#skip-name-resolve
skip-external-locking
back_log=50
max_connections=100
max_connect_errors=10
max_allowed_packet=512M
table_open_cache=1280
binlog_cache_size=2M
max_heap_table_size=384M
thread_cache_size=8
join_buffer_size=16M
sort_buffer_size=8M
key_buffer_size=256M
read_buffer_size=6M
read_rnd_buffer_size=16M
bulk_insert_buffer_size=64M
myisam_sort_buffer_size=256M
myisam_max_sort_file_size=2G
default_storage_engine=Innodb
innodb_file_per_table = 1
innodb_strict_mode=1
innodb_buffer_pool_size=1G
innodb_doublewrite=0
innodb_log_buffer_size=8M
innodb_flush_log_at_trx_commit=1
innodb_lock_wait_timeout=180
innodb_thread_concurrency=8
innodb_sync_spin_loops=100
innodb_adaptive_hash_index=ON
innodb_max_dirty_pages_pct=90
thread_stack=192K
transaction_isolation=REPEATABLE-READ
tmp_table_size=384M

secure-file-priv=""
explicit_defaults_for_timestamp=1
`
